src/__tests__/testHelpers.ts

```diff
@@ -0,0 +1,101 @@
+import * as fs from 'fs';
```

Comment:
This file is largely taken from the node repository. Future us should just export this common functionality in the common module as well.